app/tests/components/MainSection-test.js

```diff
@@ -1,43 +1,33 @@
 import React from 'react';
-import ReactTestUtils from 'react-addons-test-utils';
 import expect from 'expect';
-import { wrap } from 'react-stateless-wrapper';
+import { mount } from 'enzyme';
 import MainSection from 'components/MainSection';
-import TopicItem from 'components/TopicItem';
 
-const WrappedMainSection = wrap(MainSection);
+const emptyData = [];
+const topicItemData = [{
+  text: '',
+  id: '',
+  index: 0,
+  onIncrement: () => {},
+  onDecrement: () => {},
+  onDestroy: () => {}
+}];
+const stubFunctions = {
+  onIncrement: () => {},
+  onDecrement: () => {},
+  onDestroy: () => {}
+};
 
-describe('MainSection', () => {
-  let result;
-  let topicItems;
-  const topicItemData = {
-    text: '',
-    id: '',
-    index: 0,
-    onIncrement: () => {},
-    onDecrement: () => {},
-    onDestroy: () => {}
-  };
-  const stubFunctions = {
-    onIncrement: () => {},
-    onDecrement: () => {},
-    onDestroy: () => {}
-  };
-  const topics = [topicItemData];
-
-  describe('Has topics', () => {
-    it('should render TopicItems', () => {
-      result = ReactTestUtils.renderIntoDocument(<WrappedMainSection topics={topics} {...stubFunctions} />);
-      topicItems = ReactTestUtils.scryRenderedComponentsWithType(result, TopicItem);
-      expect(topicItems.length).toBe(1);
+describe('<MainSection />', () => {
+  describe('With Topics', () => {
+    it('should render <TopicItem> list items', () => {
+      expect(mount(<MainSection topics={topicItemData} {...stubFunctions} />).find('.list li').length).toBe(1);
```

Comment:
I'd still go with `find` , the way you did it, following [this](https://github.com/airbnb/enzyme/blob/master/docs/api/shallow.md) doc here in enzyme.

so instead of using `.list li`

I would 
```javascript
expect(mount(<MainSection topics={topicItemData} {...stubFunctions} />).find(TopicItem).length).toBe(1);
```

>  is there any documentation that you would like me to add/update?

So far we don't have any docs for testing yet, but I think we should outline good testing practices here as a doc after this gets merged in.

Thanks!